### PR TITLE
Derek Cruz | prep-javascript-arrays.

### PR DESCRIPTION
**Pull request by Derek-Cruz**
Array

### CODE REVIEW

File: prep-javascript-arrays/main.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+console.log(languagesArray);
+
+languagesArray.unshift('C++');
+console.log(languagesArray);
+
+languagesArray.shift();
+console.log(languagesArray);
+
+var thirdElement = ['javascript'];
+console.log('3rd element in languagesArray', thirdElement);
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js]: Use the index to get the third element like this: `languagesArray[2]`.

File: prep-javascript-arrays/main.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+var arraylength = languagesArray;
```
Comment (thebearingedge) [inline prep-javascript-arrays/main.js]: This would be `var arrayLength = languagesArray.length;`

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Nice! Just a couple small changes.

**Comment by Derek-Cruz** [comment]
arrays-fix

**Review by thebearingedge — APPROVED** [review]
Approved!

**Pull request merged**